server/mailet/mailetcontainer-camel/src/test/java/org/apache/james/mailetcontainer/lib/AbstractStateMailetProcessorTest.java

```diff
@@ -211,7 +212,7 @@ public void afterMailet(Mailet m, String mailName, String state, long processTim
     }
 
     @Test
-    public void testMailetThrowException() throws Exception {
+    public void testMailetNoThrowException() throws Exception {
```

Comment:
That test name look possible to improve and should match our naming convention ;-)

Reply:
I will change it.